components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultAuthenticationRequestHandler.java

```diff
@@ -262,10 +263,12 @@ protected void concludeFlow(HttpServletRequest request, HttpServletResponse resp
 
             SessionContext sessionContext = null;
             String commonAuthCookie = null;
+            String sessionContextKey = null;
             if (FrameworkUtils.getAuthCookie(request) != null) {
                 commonAuthCookie = FrameworkUtils.getAuthCookie(request).getValue();
+                sessionContextKey = DigestUtils.sha256Hex(commonAuthCookie);
```

Comment:
move following logic inside of `if (commonAuthCookie != null) {`

`sessionContextKey = DigestUtils.sha256Hex(commonAuthCookie);`